core/src/com/team30/game/screen/GameScreen.java

```diff
@@ -76,10 +89,16 @@
         camera.update();
 
         // Create and move Auber to centre room
-        auber = new Auber(31, 32);
+        auber = new Auber(10, 10);
```

Comment:
We need to avoid doing this, with the hidden constants. Especially when switching between maps

Reply:
Yes,  we do. Can we get the map size and then just place the Auber directly in the middle?